### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by hexsecs**
Removed hyperbolic claims and added some proposed clarifications.

### CODE REVIEW

**Comment by jhdalek55** [comment]
I think this is fine. Thanks, Phil. I may take a look later to see if this language shows up elsewhere, on the website, but no reason to hold this up.

**Comment by jhdalek55** [comment]
I think this is good to go. I added the link for V.2.0.0.

**Comment by jhdalek55** [comment]
Can we get at least one more approval. It's rather important that we remove this hyperbolic ASAP.

**Review by pattivacek — APPROVED** [review]

**Pull request merged**
